## Yeast transformation with PEG LiAc 
This protocol is modified from the standard protocol: 

Gietz RD, Woods RA. Transformation of yeast by lithium acetate/single-stranded carrier DNA/polyethylene glycol method. Methods Enzymol. 2002;350:87-96.   
doi: 10.1016/s0076-6879(02)50957-5. PMID: 12073338.

1. Inoculation of yeast into liquid culture in YPAD the day before (overnight culture 2ml)
2. Regrown yeast in new media until log phase
   - Dilute OD from overnight culture to OD = 0.2 and grow until approximately 0.8 (about 5-6h)
   - Volume ü°∫ about 1ml per transformation (more than enough, 500 uL culture per transformation also works well)
3. Collect and wash cells 
   - Centrifugation (two big centrifuges at end of next-door lab: 3000g for 5 minutes, RT, acceleration, and deceleration = 9 )
  - Discard supernatant
   - Resuspend in the same volume (or 1/2 volume)
   - Centrifuge again
   - Discard supernatant

4. Reagents: 1M LiAc, 50% PEG 3350(stored at RT sometimes in fridge), ssDNA (salmon sperm DNA 2mg/ml, becomes single strand DNA after boiling: Eppendorf in freezer)
  - Use autoclaved Eppendorf tubes (autoclaving room) and label them
  - Boil ssDNA 5 minutes at 95¬∞C and transfer immediately to ice.
  - Resuspend yeast in 0.1M LiAc (Dilution volume = ml of initial culture volume x 20ul, H2O = 0.9 x dilution volume, LiAc = 0.1 x dilution volume)
  - Transfer to another 1.5ml Eppendorf tubes 
  - Make transformation mix: 240ul PEG + 40ul 1M LiAc + 50ul ssDNA in the labelled Eppendorf tubes, mix 
  - Add the plasmids ü°∫ around 800ng each
  - Add 20ul of diluted cells and vortex.
  - Incubate for 15min at 28¬∞C shaking incubator (use floating Styrofoam to place eppis)
  - Pre-heat water bath (room next door, next to window) at 42¬∞C
  - Put in water bath (not the metal one) for 1 hour at 42¬∞C, shake it frequently to avoid precipitation or clotting.
  - Collect cells by spinning max speed for 1-2 minutes, discard supernatant
  - Dilute according to plate size 
  - Plate 1:10 of the cells (big plate = up to 100ul, small plate 50ul)







## Transformation into E. Coli (¬ª1hour)
 
1. 	Take eppis with 50ul E. Coli culture (main lab, -80¬∞C, bottom middle, DH10B, red lid).
2. 	Thaw on ice (use white box, ice machine is in the room that connects two hallways, after the PCR room)
3. 	Use 1:10 volume for unpurified plasmid (e.g. straight after golden gate reaction)
¬∑  	5ul DNA in 50ul E. Coli
4. 	Leave cells on ice for 20 minutes.
5. 	Pre-heat heat block or water bath to 42¬∞C (next to window in next door lab)
6. 	Heat shock at 42¬∞C for 45 seconds.
7. 	Put back on ice for one to 5 minutes.
8. 	Add 500ul LB the eppis.
9. 	Incubate for 30 minutes (in the 37¬∞C incubator use the floaty Styrofoam to hold the eppis)
10.  Plate 100ul on selection media.
11.  Label plate at the bottom and put them at 37¬∞C upside-down in the incubator.


## Y2H plate assay ‚Äì the quicker version
1. Have the plates with the transformed yeast ready, they should have nice colonies.
2. Take a 96-well PCR plate and add 30-40 ŒºL of MQ water to each well, with one well per sample. Save yourself time and energy by using a multi-channel which can be found either at the incubators in our room, or in the middle of the next room, close to the door.
3. For each plate take a tip and pick around 4-5 medium sized colonies (all of them on the same tip). The number of colonies depends on the volume of water used so for 30-40 ŒºL, 4-5 is enough.
4. Resuspend the picked colonies in the PCR plate wells. Make sure to resuspend it well here so as not to have to big clumps (this could be a problem for getting a good OD) measurement, the suspension in the well should become cloudy after this.
5. With this your yeast are ready, and now you have to readjust the OD600 to around 0.2. It doesn‚Äôt have to be 0.200, but be in the +-0.01 range. This will be done using the Nanodrop.
6. Open the Nanodrop program and pick the cell suspension setting (should be in the bottom part of the menu). When there set the value called ‚ÄúUser cursor‚Äù to 600 nm. Also make sure to adjust the maximum value shown to around 0.3 if you want to see the spectra.
7. Blank with MQ water and measure the OD600 normally ‚Äì 1.5 ŒºL for the measurement. If the OD600 isn‚Äôt in the acceptable window, add MQ water until it is. Usually if you are above 0.261 you can add 10 ŒºL and see where it goes from there. In case you go too low, resuspend additional colonies. My advice is to add less rather than more, it will result in less readjustments in total.
8. Once you have all of the yeast at the appropriate OD600 value they are ready to be plated.
9. For the reverse Y2H we use SD -leu -trp -his plates supplemented with 0.1% 5-FOA (this you can find in the IDEC freezer, small dark bottle in a bubble wrap bag, the 100x on it is correct so you need to dilute it 100 times in the medium). This can be found in the reverse double Y2H paper.
10. To plate the yeast you need to prepare serial dilutions, usually the cell suspension we prepared and then 3 decimal dilutions. Again, save yourself time using a multichannel. 
11. Once all the dilutions are prepared, plate the yeast using a multichannel, plating 5 ŒºL of each sample. Make sure that all the samples are aligned, you can position the PRC plate bellow the medium plate to act as a guide.
12. Once the yeast are plated, let the plates dry, collect and grow in a static incubator at 28 ¬∞C for 2-3 days. 

## Transformation to Agrobacteria



1. Cool down cuvettes in freezer (next door lab, under the middle window, drawer marked ‚ÄúCUVETTES‚Äù or clean ones on iDEC bench)
2. Take Agrobacteria out of -80¬∞C freezer on to ice ‚Üí middle drawer of lowest row, box that is labelled with GV3101 (not anything else additionally!)
3. Get electroporation cuvettes should be pre-cooled in 4 or -20¬∞C 
4. Add all of agrobacteria into slit in cuvette
5. Add 1.5 uL (+/- 100ng) of purified plasmid on top, do not mix!
6. Prepare tubes with 500 uL of LB
7. Go to electroporation machine (to the right of the spectrophotometer), turn on and set to 1.4 kV
8. Put cuvette in holder (make sure orientation is correct) and push into machine until it clicks
9. Press both red buttons until you hear a beep, the machine should show less than 5 ms (or slightly above) otherwise you have to repeatBe careful, if cuvette is not clean the machine will produce a lighting bolt
10. Directly after electroporation, add 500 uL LB to the cuvette and resuspend.
11. Transfer back to tube.
12. Incubate at 28¬∞C for 2h.
13. Plate cells on selection media (for GV3101 Rif/Gen, for pTRV2 Kan ‚Üí LB-Rif/Gen/Kan).
14. Incubate at 28¬∞C.
15. Rinse cuvettes well with tap water and then leave them in a beaker of ethanol for a while and then dry them.

## Plant Transformation Using Agrobacterium Protocol


Materials:
- Agrobacterium strain (construct + p19)
- Infiltration buffer: Magnesium Chloride (MgCl2) and MES
- AS (Acetosyringone) in DMSO
- Centrifuge
- Syringe (1 ml)
- Spectrophotometer (for measuring OD)

Procedure:

1. Agrobacterium Culture Preparation:
   - Grow the Agrobacterium containing your desired construct.
   - Grow the p19 Agrobacterium, which is cotransformed to suppress the plant defense response.

2. Centrifugation:
   - Spin down the Agrobacterium culture for 10 minutes at 3,000 G.

3. Preparation of Infiltration Buffer:
   - Prepare a stock solution with final concentrations of 10 mM for both Magnesium Chloride (from a 100 mM stock, dilute 100 times) and MES (from a 500 mM stock, dilute 50 times).
   - Acetylsaringone (AS) stock is at a concentration of 0.1 M
      - Either dilute the AS stock in DMSO or use a prepared stock solution from the freezer.
      - Add between 150-200 Œºl of AS to every 100 ml of the infiltration buffer.

4. Resuspension:
   - Resuspend the Agrobacterium pellet from step 2 in 5 ml of the infiltration buffer.
   - Centrifuge for 10 minutes at 3,000 G.
   - Resuspend the pellet in the final infiltration buffer containing both the buffer and AS.

5. OD Adjustment:
   - Adjust the Agrobacterium culture to a final optical density (OD) of 0.5.
      => construct should represent an OD of 0.4 while P19 should represent 0.1 OD.
   
      - Option 1: Adjust all concentrations to an OD of 0.5. Mix at a ratio of 1:5 for P19 and 4:5 for the construct.
      - Option 2: Directly measure and adjust the concentrations to achieve 0.4 OD for the construct and 0.1 OD for P19. Dilute using the infiltration buffer.
=> final volume 10ml
=> Keep a separate 0.5 OD of P19 as a negative control
7. Incubation:
   - Incubate the Agrobacterium culture on a shaker for 1 hour. The shaker is located in the lab next door, against the wall of our lab.

8. Infiltration:
   - Retrieve a 1 ml syringe (from the next door lab on the corner shelf)
   - Label plant leaves 
   - Fill the syringe with the Agrobacterium culture.
   - Press the syringe into the underside of the plant leaf while using a finger to press against the top side of the leaf for efficient infiltration.

## Golden Gate cloning 

The enzymes are found in the yellow
Use online molar ratio calculator:
Insert bp and concentration ‚áí molar concentration
|           | 10ul system   | 20 ul system  |
| --- | --- | --- |
| backbone             | 1ul | 2ul |
| insert  | calculate ratio  | calculate ratio |
| 10 x T4 ligase buffer| 1ul  | 2ul
| T4 ligase | 0.5ul | 1ul |
| BpiI/Esp3I/BsaI | 0.5ul | 1ul |
| ddH2O  | 10-insert | 20-insert |


 ‚ÄúGolden Gate Box‚Äù in the freezer next-door
We want +/- 3 times insert per backbone molar ratio (the backbone is always 1ul)



 Add to DIGLIG thermocycler (if in use, also other Thermal cyclers available):
File ü°∫ DIGLIG
Change volume accordingly
The only thing you realistically change is the number of cycles, if something is efficient and you want it done quickly, you are looking at around 15 cycles, and if not then go for 28ish 




## Digestion/Ligation 

 
|  | 20ul System |
| --- | --- |
| DNA | >2ug (200ng) |
| Enzyme 1 | 1.5ul |
| Enzyme 2 | 1.5ul |
|10x CutSmart Buffer | 2ul |
| ddH2O | Fill up to 20ul |



 
Digestion:
¬∑  	First calculate DNA and water volume
¬∑  	If DNA volume exceeds the 20ul of the system, increase total system accordingly
¬∑  	The enzymes can be found in the freezer of the next-door lab top left
¬∑  	Add the enzymes last and keep on cold block
 
¬∑  	Incubate for 2 hours or overnight at 37¬∞C
¬∑  	Clean-up the product before starting the ligation
 
 
 
## ROS burst assay

1. Preparation of Leaf Disks:
   - Cut leaf disks from Nicotiana benthamiana plants.
2. Preparation of Chemiluminescent Reagent:
   - Prepare a solution containing 10 ¬µM LO-12, 10 ¬µg/ml  of horseradish peroxidase and 100ug/ml of chitin in water. Add 100 ¬µl of this solution per leaf disk. 
3. ROS Elicitation: 
  - remove the water from the container without disturbing the leaf disks
  - Replace the water with 100 ¬µl of the prepared chemiluminescent reagent with chitin per leaf disk.
   - Negative Control: Prepare a negative control by treating additional leaf disks with 100 ¬µl of water (without any PAMPs or chemiluminescent reagent).
4. Measurement of Luminescence: 
  - Place the container in the Tecan spark  machine  and measure luminescence for around 45  minutes. Record the data as total counts over the 45-minute period. 


## Flow Cytometry 

1. Colony Preparation:
  - Select six colonies from each group.
  - Resuspend each colony in 1.5 mL of yeast SD (Synthetic Defined) medium.
2. Overnight Culturing:
  - Transfer the resuspended colonies to a 96-deep well plate (Corning¬Æ).
  - Seal the plate with 3M‚Ñ¢ Micropore‚Ñ¢ Surgical Tape.
  - Incubate overnight to allow for culture growth.
  3. Sample Preparation:
  - Following overnight incubation, dilute the cultures 5-fold using PBS (Phosphate-buffered saline) buffer.
  4. Flow Cytometry Measurement:
  - Load the diluted samples into the flow cytometer.
  - Follow the flow cytometry protocol for yeast fluorescence measurement as described in Ottoz et al., 2016.
5. Data Analysis:
  - Analyze the collected data using FlowJo software.
  - Perform a one-way ANOVA with Dunnett‚Äôs posthoc test to assess the differences between sample fluorescence means.


## Error-Prone PCR Protocol:

1. Polymerase Preparation: Prepare GoTaq polymerase according to the corresponding protocol provided by the manufacturer. 
2. Mutation Rate Calculation: Calculate the mutation rates using the Thermofisher fidelity calculator to predict a single point mutation across the CERK1 sequence. 
3. PCR Setup: Adjust the annealing temperatures according to the primers being used. Set up the thermocycler to run for 35 cycles. 
4. PCR Execution: Load the prepared mixture into the thermocycler and run the PCR. 
5. Product Purification: Purify the PCR product following standard purification protocols. 




## Virus-Induced Gene Silencing (VIGS) Protocol: 

1. Agrobacteria Transformation: Transform electrocompetent agrobacteria as previously described with either pTRV1 or pTRV2-CERK1 vectors. Culture Preparation: Recover and culture the transformed bacteria overnight as previously described. 
2. Culture Adjustment: The following day, adjust the cultures to an OD600 of 2.0. 
3. Culture Mixing: Mix pTRV1 and pTRV2-CERK1 cultures in a 1:1 ratio. 
4. Incubation: Incubate the mixture for 1-3 hours at room temperature with shaking. 
5. Plant Infiltration: Infiltrate the true leaves of 1-week-old N. benthamiana plants with the culture mixture using a syringe. 
6. Growth: Grow the infiltrated plants for 3-4 weeks under greenhouse conditions. 
7. Validation: Perform a ROS burst assay to validate the gene silencing.
