
<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../background/">
      
      
        <link rel="next" href="../Outlooks/">
      
      
      <link rel="icon" href="../../img/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.4.4">
    
    
      
        <title>Results - Evolution Suisse</title>
      
    
    
      <link rel="stylesheet" href="../../assets/stylesheets/main.bd3936ea.min.css">
      
        
        <link rel="stylesheet" href="../../assets/stylesheets/palette.356b1318.min.css">
      
      


    
    
      
    
    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Urbanist:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Urbanist";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../../stylesheets/extra.css">
    
    <script>__md_scope=new URL("../..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
   <link href="../../assets/stylesheets/glightbox.min.css" rel="stylesheet"/><style>
        html.glightbox-open { overflow: initial; height: 100%; }
        .gslide-title { margin-top: 0px; user-select: text; }
        .gslide-desc { color: #666; user-select: text; }
        .gslide-image img { background: white; }
        
            .gscrollbar-fixer { padding-right: 15px; }
            .gdesc-inner { font-size: 0.75rem; }
            body[data-md-color-scheme="slate"] .gdesc-inner { background: var(--md-default-bg-color);}
            body[data-md-color-scheme="slate"] .gslide-title { color: var(--md-default-fg-color);}
            body[data-md-color-scheme="slate"] .gslide-desc { color: var(--md-default-fg-color);}
            </style> <script src="../../assets/javascripts/glightbox.min.js"></script></head>
  
  
    
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="custom" data-md-color-accent="green">
  
    
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
        
        <a href="#achievements" class="md-skip">
          Skip to content
        </a>
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href="../.." title="Evolution Suisse" class="md-header__button md-logo" aria-label="Evolution Suisse" data-md-component="logo">
      
  <img src="../../img/Logo.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Evolution Suisse
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Results
            
          </span>
        </div>
      </div>
    </div>
    
      
    
    
    
    
      <div class="md-header__source">
        <a href="https://github.com/idec-teams/2023_Evolution_Suisse" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../.." class="md-tabs__link">
        
  
    
  
  Overview

      </a>
    </li>
  

      
        
  
  
    
  
  
    
    
      <li class="md-tabs__item md-tabs__item--active">
        <a href="../background/" class="md-tabs__link">
          
  
  Project

        </a>
      </li>
    
  

      
        
  
  
  
    
    
      <li class="md-tabs__item">
        <a href="../../documentation/protocols/" class="md-tabs__link">
          
  
  Documentation

        </a>
      </li>
    
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../../team/" class="md-tabs__link">
        
  
    
  
  Team

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href="../.." title="Evolution Suisse" class="md-nav__button md-logo" aria-label="Evolution Suisse" data-md-component="logo">
      
  <img src="../../img/Logo.png" alt="logo">

    </a>
    Evolution Suisse
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/idec-teams/2023_Evolution_Suisse" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"/></svg>
  </div>
  <div class="md-source__repository">
    GitHub
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../.." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Overview
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--active md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_2" checked>
        
          
          <label class="md-nav__link" for="__nav_2" id="__nav_2_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Project
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_2_label" aria-expanded="true">
          <label class="md-nav__title" for="__nav_2">
            <span class="md-nav__icon md-icon"></span>
            Project
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../background/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Background
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
    
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
        <label class="md-nav__link md-nav__link--active" for="__toc">
          
  
  <span class="md-ellipsis">
    Results
  </span>
  

          <span class="md-nav__icon md-icon"></span>
        </label>
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Results
  </span>
  

      </a>
      
        

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#achievements" class="md-nav__link">
    Achievements
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#results" class="md-nav__link">
    Results
  </a>
  
    <nav class="md-nav" aria-label="Results">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#avrptob-interacts-with-cerk1-and-impairs-cerk1-mediated-immune-responses" class="md-nav__link">
    AvrPtoB interacts with CERK1 and impairs CERK1-mediated immune responses
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#development-of-trust-ry2h-a-novel-reverse-yeast-two-hybrid-system" class="md-nav__link">
    Development of TRUST-rY2H, a novel reverse yeast two-hybrid system
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#disruption-of-cerk1-avrptob-interaction-using-trust-ry2h" class="md-nav__link">
    Disruption of CERK1-AvrPtoB interaction using TRUST-rY2H
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#in-silico-prediction-of-interaction-disrupting-mutations" class="md-nav__link">
    In silico prediction of interaction-disrupting mutations
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#in-planta-screening-of-avrptob-resistant-cerk1-variants-that-retain-kinase-activity-via-transient-expression-in-n-benthamiana" class="md-nav__link">
    In planta screening of AvrPtoB-resistant CERK1 variants that retain kinase activity via transient expression in N. benthamiana
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#development-of-trust-rmyth-a-reverse-split-ubiquitin-based-membrane-protein-yeast-two-hybrid-system-with-e2a-bleor-anti-truncation-strategy" class="md-nav__link">
    Development of TRUST-rMYTH, a reverse split ubiquitin-based membrane protein yeast two-hybrid system with E2A-BleoR anti-truncation strategy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#construction-of-fluorescent-reporter-strains-to-quantitatively-measure-interaction-strength" class="md-nav__link">
    Construction of fluorescent reporter strains to quantitatively measure interaction strength
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    References:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
      
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../Outlooks/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Outlook
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../references/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    References
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
    
    
    
    
    
      
      
    
    <li class="md-nav__item md-nav__item--section md-nav__item--nested">
      
        
        
        
        <input class="md-nav__toggle md-toggle " type="checkbox" id="__nav_3" >
        
          
          <label class="md-nav__link" for="__nav_3" id="__nav_3_label" tabindex="">
            
  
  <span class="md-ellipsis">
    Documentation
  </span>
  

            <span class="md-nav__icon md-icon"></span>
          </label>
        
        <nav class="md-nav" data-md-level="1" aria-labelledby="__nav_3_label" aria-expanded="false">
          <label class="md-nav__title" for="__nav_3">
            <span class="md-nav__icon md-icon"></span>
            Documentation
          </label>
          <ul class="md-nav__list" data-md-scrollfix>
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../documentation/protocols/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Protocols
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../documentation/notebook/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Lab Notebook
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../files/report.pdf" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Download Report
  </span>
  

      </a>
    </li>
  

              
            
              
                
  
  
  
    <li class="md-nav__item">
      <a href="../../files/supplement.pdf" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Download Supplimentary Data
  </span>
  

      </a>
    </li>
  

              
            
          </ul>
        </nav>
      
    </li>
  

    
      
      
  
  
  
    <li class="md-nav__item">
      <a href="../../team/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Team
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
    <label class="md-nav__title" for="__toc">
      <span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
    <ul class="md-nav__list" data-md-component="toc" data-md-scrollfix>
      
        <li class="md-nav__item">
  <a href="#achievements" class="md-nav__link">
    Achievements
  </a>
  
</li>
      
        <li class="md-nav__item">
  <a href="#results" class="md-nav__link">
    Results
  </a>
  
    <nav class="md-nav" aria-label="Results">
      <ul class="md-nav__list">
        
          <li class="md-nav__item">
  <a href="#avrptob-interacts-with-cerk1-and-impairs-cerk1-mediated-immune-responses" class="md-nav__link">
    AvrPtoB interacts with CERK1 and impairs CERK1-mediated immune responses
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#development-of-trust-ry2h-a-novel-reverse-yeast-two-hybrid-system" class="md-nav__link">
    Development of TRUST-rY2H, a novel reverse yeast two-hybrid system
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#disruption-of-cerk1-avrptob-interaction-using-trust-ry2h" class="md-nav__link">
    Disruption of CERK1-AvrPtoB interaction using TRUST-rY2H
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#in-silico-prediction-of-interaction-disrupting-mutations" class="md-nav__link">
    In silico prediction of interaction-disrupting mutations
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#in-planta-screening-of-avrptob-resistant-cerk1-variants-that-retain-kinase-activity-via-transient-expression-in-n-benthamiana" class="md-nav__link">
    In planta screening of AvrPtoB-resistant CERK1 variants that retain kinase activity via transient expression in N. benthamiana
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#development-of-trust-rmyth-a-reverse-split-ubiquitin-based-membrane-protein-yeast-two-hybrid-system-with-e2a-bleor-anti-truncation-strategy" class="md-nav__link">
    Development of TRUST-rMYTH, a reverse split ubiquitin-based membrane protein yeast two-hybrid system with E2A-BleoR anti-truncation strategy
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#construction-of-fluorescent-reporter-strains-to-quantitatively-measure-interaction-strength" class="md-nav__link">
    Construction of fluorescent reporter strains to quantitatively measure interaction strength
  </a>
  
</li>
        
          <li class="md-nav__item">
  <a href="#references" class="md-nav__link">
    References:
  </a>
  
</li>
        
      </ul>
    </nav>
  
</li>
      
    </ul>
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


<h2 id="achievements">Achievements</h2>
<p>Key achievements:</p>
<ul>
<li>
<p>Pioneered using directed evolution technology to create disease resistant crops</p>
</li>
<li>
<p>Proposed a novel strategy to enhance plant immunity by engineering plant pattern recognition receptors to elude attacks by effectors</p>
</li>
<li>
<p>Developed TRUST-rY2H, a novel reverse yeast two-hybrid system (rY2H) to disrupt protein-protein interactions.</p>
</li>
<li>
<p>Identified mutations disrupting CERK1-AvrPtoB interaction using TRUST-rY2H and machine learning-based computational design.</p>
</li>
<li>
<p>Designed a screening method to rapidly test the activity and AvrPtoB resistance of CERK1 variants in plants.</p>
</li>
</ul>
<p>We also:</p>
<ul>
<li>
<p>Developed a reverse assay for split-ubiquitin membrane protein yeast two-hybrid (MYTH) to disrupt the interaction between membrane proteins.</p>
</li>
<li>
<p>Developed fluorescent reporters for conventional Y2H and MYTH to precisely measure the strength of protein-protein interactions.</p>
</li>
</ul>
<h1 id="results">Results</h1>
<h2 id="avrptob-interacts-with-cerk1-and-impairs-cerk1-mediated-immune-responses">AvrPtoB interacts with CERK1 and impairs CERK1-mediated immune responses</h2>
<p>Effectors often target PRRs, the CERK1-AvrPtoB interaction seems to be an exception. To be able to disrupt the interaction, we first need to verify that AvrPtoB indeed targets CERK1 and analyse the effect of AvrPtoB on the CERK1 mediate immune response.</p>
<p><strong>Aim:</strong> Prove AvrPtoB targets CERK1 and leads to impaired immune response.
A Y2H plate assay was performed using AvrPtoB and CERK1 fused to both DBD and AD interchangeably (<strong>Fig. 1</strong>). In this assay, cells grow in a selective medium (-LWU) if an interaction between the proteins is present. The experiment group of DBD-CERK1 and AD-AvrPtoB showed high levels of growth, superseding even the established positive control protein pair (Antigen T and p53). This confirms a strong interaction between our two proteins of interest. </p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_1.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure1" src="../Result_figures/Figure_1.png" /></a></p>
<figcaption> Figure 1. CERK1 and AvrPtoB strongly interact in the Y2H assay only when fused to AD and DBD, respectively.</figcaption>
</figure>
<p>To follow this up, we wanted to ensure that AvrPtoB did indeed impair the immune response of plants. We treated plants with chitin to simulate a fungal infection, triggering the CERK1-mediated immune response. The intensity of the immune response can be quantified by measuring the reactive oxygen species (ROS) burst following the chitin treatment (<strong>Fig. 2</strong>). When we transiently transformed the plants with an estradiol-inducible AvrPtoB DNA construct (XVE/OlexA-AvrPtoB) and the treated the plants with estradiol, the ROS burst was significantly reduced, indicating that AvrPtoB indeed impairs CERK1’s functions.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_2.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure2" src="../Result_figures/Figure_2.png" /></a></p>
<figcaption>Figure 2.  Transient expression of AvrPtoB attenuates the chitin-triggered ROS burst in N. benthamiana. </figcaption>
</figure>
<p><strong>Conclusion:</strong> AvrPtoB binds CERK1 and impairs CERK1-mediated immune responses. However, the fusion of DBD and AD in the Y2H seems to affect the interaction. This can be seen as in one sense, the interaction is strong, and in the other sense there is little growth compared to negative controls. This indicates a false negative, and a need for a more robust system.</p>
<h2 id="development-of-trust-ry2h-a-novel-reverse-yeast-two-hybrid-system">Development of TRUST-rY2H, a novel reverse yeast two-hybrid system</h2>
<p>Original rY2H methods are unable to distinguish between informative interaction-disruption mutations, and false positives resulting from truncating mutations. Unfortunately, the systems that do resist truncations, such as the RD2H, are rigid in their design, not allowing the exchange of DBD and AD between the proteins of interest. This lack of flexibility can be detrimental when analysing certain protein-protein interactions (PPI), as the DBD or AD can hinder the interaction, leading to false-negative results as already seen in our initial experiments.</p>
<p><strong>Aim:</strong> Develop a novel rY2H system, capable of selecting against truncating mutations and flexible with AD and DBD fusions, to disrupt the CERK1-AvrPtoB interaction.</p>
<p>As mentioned, the interaction between DBD-CERK1 and AD-AvrPtoB cannot be seen when DBD and AD are switched between the proteins. Current systems for truncation-resistant Y2H mandate that the protein undergoing mutagenesis is fused to AD. This meant we would not be able to test CERK1 for truncation within current systems. To solve this problem, we devised a system that would fuse BleoR, a gene conferring resistance to the antibiotic zeocin, to CERK1 variants. This would mean that BleoR would only be expressed if the CERK1 variant was complete, i.e. not truncated, and therefore the cells would be able to grow in zeocin-containing media (<strong>Fig. 3</strong>).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_3.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure3" src="../Result_figures/Figure_3.png" /></a></p>
<figcaption> Figure 3. Schematic explanation of the TRUST-rY2H system (upper panel) and its behaviour depending on the interaction state and selective media used (lower panels).</figcaption>
</figure>
<p>We originally fused the BleoR to CERK1 by using a (GS)<sub>3</sub> linker, a long stretch of glycine and serine residues, but this approach disrupted both the CERK1-AvrPtoB interaction, as well as BleoR anti-zeocin activity. This mutual interference was almost certainly due to the proximity of BleoR and the CERK1-AvrPtoB interaction complex. We hypothesised that this issue could be solved by using a self-cleaving peptide, E2A, which would separate CERK1 and BleoR once they had been translated (see <strong>Fig. 3</strong> for a schematic representation). A rY2H assay (<strong>Fig. 4</strong>) shows that the version with the (GS)<sub>3</sub> linker, compared to the one using E2A, clearly exhibited worse growth on -Ura and +Zeocin plates, indicating the lack of CERK1-AvrPtoB interaction and BleoR activity, respectively. This confirmed that using E2A considerably reduced the interference BleoR and the CERK1-AvrPtoB complex had on each other. </p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_4.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure4" src="../Result_figures/Figure_4.png" /></a></p>
<figcaption> Figure 4. Using an E2A self-cleaving peptide instead of the (GS)<sub>3</sub> linker reduces the negative impact the BleoR anti-truncation system has on the CERK1-AvrPtoB interaction.</figcaption>
</figure>
<p>We also tested whether other self-cleaving peptides, O2A and P2A, could do a better job. However, we found E2A to be the most effective (<strong>Fig. 5</strong>).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_5.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure5" src="../Result_figures/Figure_5.png" /></a></p>
<figcaption>Figure 5. Compared to other self-cleaving peptides, the E2A peptide causes minimal disruption of the CERK1-AvrPtoB interaction.
 </figcaption>
</figure>
<p>We also used a fluorescent reporter strain(see last section for more info) to precisely quantify the effects of different 2A peptides on CERK1-AvrPtoB interaction (<strong>Fig. 6</strong>). We got similar results as the plate assay, indicating that E2A is the best option.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_6.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure6" src="../Result_figures/Figure_6.png" /></a></p>
<figcaption>Figure 6. Differences between sample means in the flow cytometry data were assessed using one-way ANOVA with Dunnett’s posthoc test; &lt;0.01 (*), &lt;0.0001(****). </figcaption>
</figure>
<p>Lastly, we wanted to test whether the E2A-BleoR anti-truncation strategy was capable of selecting against truncations in practice. To do this, we manually introduced a stop codon, as well as a 1-bp deletion to the DNA sequence CERK1. </p>
<p>Initially, we did a Y2H plate assay starting from colonies which have been grown for &gt;48h post-transformation (<strong>Fig. 7</strong>). Unfortunately, the result shows that our system wasn’t capable of successfully selecting against truncation mutations, as evidenced by yeast growth on zeocin-containing plates.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_7.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure7" src="../Result_figures/Figure_7.png" /></a></p>
<figcaption> Figure 7. The addition of E2A-BleoR is not sufficient to select against truncating mutations when colonies were selected for more than  48h post-transformation.</figcaption>
</figure>
<p>We then redid the assay by plating the yeast that have just been transformed with the plasmids harbouring the Y2H system, as would be done in our directed evolution workflow (<strong>Fig. 8</strong>). This time, the truncated mutant showed notably less growth, implying that our system works. This is perhaps due to the leaky expression of BleoR, which might be caused by sequences in CERK1 which resemble canonical translation start sites (Monteuuis et al., 2019). Over time, the leaky expression could lead to the accumulation of BleoR, allowing even the truncated mutants to grow on zeocin-containing plates. To overcome this, we plan to introduce D25A mutation to BleoR to compromise its activity by ~ 50% (Dumas et al., 1994), which has been proven useful in counteracting leaky expression (Tominaga et al., 2021). </p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_8.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure8" src="../Result_figures/Figure_8.png" /></a></p>
<figcaption>Figure 8. When plated immediately post-transformation, truncation mutations are selected against by the BleoR anti-truncation system. </figcaption>
</figure>
<p>We coined this system TRUST-rY2H, a reverse yeast two-hybrid (rY2H) system that is Truncation-Resistant and Universal, using Self-cleavage peptide Technology (TRUST). We summarised the comparison of TRUST-rY2H to some of existing rY2H methods in Table 1.</p>
<p><strong>Table 1.</strong> Comparison of TRUST-rY2H to previous methods </p>
<table>
<thead>
<tr>
<th></th>
<th>System avoids truncations</th>
<th>Single step selection</th>
<th>Protein of interest can be fused to both the activating domain and DNA binding  domain</th>
<th>Compatible with membrane systems (split ubiquitin system)</th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Original rY2H</strong></td>
<td>-</td>
<td>-</td>
<td>-</td>
<td>+</td>
</tr>
<tr>
<td><strong>rY2H</strong>(Vidal et al.)</td>
<td>+</td>
<td>-</td>
<td>+</td>
<td>-</td>
</tr>
<tr>
<td><strong>rY2H</strong> (Leanna et al.)</td>
<td>-</td>
<td>-</td>
<td>+</td>
<td>-</td>
</tr>
<tr>
<td><strong>rDY2H</strong> (Vincent et al.)</td>
<td>+</td>
<td>+</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td><strong>One-plus Two Hybrid system</strong>(Kim et al.)</td>
<td>+</td>
<td>-</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td><strong>TRUST-rY2H</strong>(Kim et al.)</td>
<td>+</td>
<td>+</td>
<td>+</td>
<td>+</td>
</tr>
</tbody>
</table>
<p><strong>Conclusion:</strong> Fusing BleoR to proteins along with a self-cleaving peptide has minimal effect on the protein interaction and could, with further optimisation, provide adequate truncation resistance.</p>
<h2 id="disruption-of-cerk1-avrptob-interaction-using-trust-ry2h">Disruption of CERK1-AvrPtoB interaction using TRUST-rY2H</h2>
<p>The TRUST-rY2H is ready. Now it’s time to use it to achieve our goal, disrupting the CERK1-AvrPtoB interaction.</p>
<p><strong>Aim:</strong> Use TRUST-rY2H to disrupt CERK1-AvrPtoB interaction.</p>
<p>We performed mutagenesis on CERK1 using error-prone PCR, and subjected the mutagenesis library to TRUST-rY2H to select for mutations that disrupt CERK1-AvrPtoB interaction. For comparison, we also run a classic rY2H in parallel, which is without an anti-truncation system. Ten colonies from each screen were sequenced to identify mutations (Table 2). Three out of ten colonies from TRUST-rY2H were false positives with nonsense mutations (stop codons), while the classic rY2H screen produced two such false positives. Interestingly, only two mutations (S444F and G461R) were located in or close to the predicted interaction interface. Other mutations were dispersed throughout the whole protein sequence, suggesting a diverse array of mutations impacting the interaction beyond the predicted interaction surface.</p>
<p><strong>Table 2.</strong> Summary of CERK1 variants obtained from the reverse yeast two-hybrid assays. Each line shows mutations found in a single epPCR colony. Notation R415G means R to G substitution in 415th amino acid. Star denotes a stop codon. Mutations in the interaction regions are underlined.  </p>
<table>
<thead>
<tr>
<th><strong>Classic rY2H without anti-truncation</strong></th>
<th><strong>TRUST-rY2H</strong></th>
</tr>
</thead>
<tbody>
<tr>
<td>S510P <br> L398P <br> G502E <br> S444F <br> G461R <br> F267L <br> L363Q <br> G569R <br> C381W</td>
<td>R415G <br> A521T <br> Y337C <br> F533S <br> F359S <br> K349R</td>
</tr>
<tr>
<td>W412<em> <br> L388</em></td>
<td>K495R <br> K345<em> <br> Q451</em> <br> E575E <br> K364*</td>
</tr>
</tbody>
</table>
<p>It seems that TRUST-rY2H underperforms the classic rY2H, however, it is noteworthy that the truncation rate in the classic rY2H (20%) is far lower than reported (&gt;97%) (Barr et al., 2004, Gray et al., 2007). One possibility is that some of the hits are false positives. This can be because of mutations disrupting DBD, promoter, and <em>Ura3</em> gene, which can also lead to a 5-FOA-resistant phenotype. Therefore, we need to confirm these mutations by performing a round of forward Y2H. Another possibility is that CERK1-AvrPtoB is sensitive to mutations, i.e., among all the potential interaction-disruption mutations, there are many more non-truncation mutations than truncation mutations. Such results could also be due to stochastic colony picking and require more colonies to achieve statistical robustness.</p>
<p><strong>Conclusion:</strong> We successfully identified mutations on CERK1 that disrupt its interaction with AvrPtoB using both TRUST-rY2H and classic rY2H. We need to sequence and validate more colonies to statistically compare the performance of TRUST-rY2H and classic rY2H.</p>
<h2 id="in-silico-prediction-of-interaction-disrupting-mutations"><em>In silico</em> prediction of interaction-disrupting mutations</h2>
<p>New machine-learning-based technologies are at the forefront of PPI predictions and simulations. We decided to use this to our advantage by predicting the CERK1-AvrPtoB interaction to identify key amino acids that could be mutated in a site-directed manner to disrupt the interaction.</p>
<p><strong>Aim: </strong>Predict the interaction between CERK1 and AvrPtoB to identify potential mutations to disrupt interaction</p>
<p>Using AlphaFold-Multimer, an artificial intelligence tool specialised for predicting protein-protein complexes, to predict the CERK1-AvrPtoB interaction at an atomic level (<strong>Fig. 9</strong> left). Having a 3D model of the CERK1-AvrPtoB interaction complex, we could start investigating the binding interface - the area on the two proteins that interact with each other. Unfortunately, the prediction obtained using AlphaFold-multimer didn’t have satisfactory model confidence, i.e., certain areas could have been predicted wrong. As all of our downstream work depends on the accuracy of the interaction complex prediction, we opted to try other prediction tools, namely ColabFold. Additionally, one of the possible culprits behind this low model confidence was long, intrinsically disordered regions in both CERK1 and AvrPtoB (<strong>Fig. 9</strong> right). These parts of the protein didn’t participate in the interaction itself and didn’t have any clear 3D structure of their own, making it difficult for the model to accurately describe them. To make sure they didn’t interfere with the protein complex prediction, we removed them for the second round of predictions. These two choices paid off in the end, as the second prediction ended up with satisfactory model confidence, allowing us to proceed with the analysis.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_9.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure9" src="../Result_figures/Figure_9.png" /></a></p>
<figcaption>Figure 9. Prediction of AvrptoB-CERK1 binding complex from AlphaFold-Multimer and Colabfold. </figcaption>
</figure>
<p>Looking at the 3D structure, we first found residues in the two proteins that interacted with each other, i.e., were at an acceptable distance from each other or had physio-chemical properties facilitating interactions, such as opposing polarities. Then, within the binding interface, we found residues in CERK1 that interacted with many residues in AvrPtoB. If we could mutate these residues, they might severely affect the stability of the interaction complex, without causing a big change to CERK1 itself. We made sure to avoid functionally important regions of CERK1, so as to keep its immune function intact. Lastly, to make sure that given mutations do not completely destabilise the global CERK1 structure, we used an additional AI tool called Protein MPNN. This multipronged approach resulted in the following seven mutations: L462S, R474E, F480K, M483R, V488K, V515M, and F525T.</p>
<p>To validate our predictions, we ran a Y2H assay in both forward and reverse directions (<strong>Fig. 10</strong>). Variants that grew on plates lacking uracil, but failed to grow those containing 5-FOA, failed to disrupt the CERK1-AvrPtoB interaction, and vice versa. Out of the seven computationally predicted mutants, five (L462S, M483R, F480K, V488K, V515M) successfully disrupted this interaction. This result highlights the importance of computational models in predicting promising site-directed mutagenesis targets.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_10.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure10" src="../Result_figures/Figure_10.png" /></a></p>
<figcaption> Figure 10. Experimental validation of in silico predicted mutations that disrupt CERK1-AvrPtoB interaction.</figcaption>
</figure>
<p><strong>Conclusion:</strong> The interaction between AvrPtoB and CERK1 was successfully predicted in silico， generating seven mutations potentially able to disrupt the interaction without affecting CERK1’s function. Y2H assay confirms that five of them can disrupt the interaction. </p>
<h2 id="in-planta-screening-of-avrptob-resistant-cerk1-variants-that-retain-kinase-activity-via-transient-expression-in-n-benthamiana"><em>In planta</em> screening of AvrPtoB-resistant CERK1 variants that retain kinase activity via transient expression in N. benthamiana</h2>
<p>Loss of interaction with AvrPtoB can be accompanied by loss of kinase activity. This concern is based on the site-directed mutagenesis analysis of AvrPtoB-BAK1 interaction, where many BAK1 mutants that lost interaction with AvrPtoB also lost their kinase activities (Cheng et al., 2011).</p>
<p>For CERK1, we found that two frequently used kinase-dead mutants, D441V and K350N, either lost or exhibited reduced interaction with AvrPtoB, respectively (<strong>Fig. 11</strong>). This result suggested that CERK1’s kinase activity and its interaction with AvrPtoB might be also closely correlated. Consequently, there was a need for a screening method, able to efficiently detect unimpaired CERK1 kinase activity.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_11.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure11" src="../Result_figures/Figure_11.png" /></a></p>
<figcaption> Figure 11. Kinase-dead CERK1 mutants lost interaction with AvrPtoB suggesting CERK1’s kinase activity and capability to interact with AvrPtoB are closely correlated.
</figcaption>
</figure>
<p><strong>Aim:</strong> Develop an efficient screening approach to check the functionality of CERK1 variants in plants.</p>
<p>To screen for CERK1 variants that retained their kinase activities, we used the fact that overexpression of functional CERK1 in Nicotiana benthamiana leads to cell death (Pietraszewska-Bogiel et al., 2013; Suzuki et al., 2018). To this end, CERK1 variants are transiently expressed in wild-type N. benthamiana under a strong and constitutive 35S promoter. Those variants whose transient expression leads to a cell death phenotype should retain the kinase activity (<strong>Fig. 12</strong>).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_12.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure12" src="../Result_figures/Figure_12.png" /></a></p>
<figcaption> Figure 12. The assay used to screen for CERK1 variants that retain kinase activity, which is based on overexpression of CERK1 variants and observation of cell death phenotype. “3 dpi” means “three days after infiltration”.
</figcaption>
</figure>
<p>We initially tested our system using the wildtype CERK1 and CERK1D441V one of the kinase-dead variants which also lost its interaction with AvrPtoB. As expected, the wildtype CERK1 showed the cell death phenotype, while a kinase-dead mutant (D441V) did not (Figure 13). Unfortunately, due to time constraints, we weren’t able to test any of the variants obtained using either the directed evolution or site-directed mutagenesis.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_13.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure13" src="../Result_figures/Figure_13.png" /></a></p>
<figcaption> Figure 134  Expression of wild-type CERK1 and not the D441V kinase-dead mutant caused cell death in N. benthamiana. (Please ignore the holes on the leaves. They are just because we took some leaf discs.)
</figcaption>
</figure>
<p>Active CERK1 variants will be subjected to a second round of in planta screening designed to screen for real AvrPtoB-resistant CERK1 variants (<strong>Fig. 14</strong>). This round of screening will be conducted using a N. benthamiana line which doesn’t produce its native CERK1 (NbCERK1). This was achieved through virus-induced gene silencing (VIGS). These silenced plants produce no CERK1, so we are able to express and test our CERK1 variants in them. Our CERK1 variants will be expressed using the native promoter (ProCERK1), ensuring a low enough expression level to avoid cell death, but high enough to result in a measurable ROS burst in response to chitin treatment. At the same time, AvrPtoB will be also expressed using an estradiol-inducable promoter. By comparing the chitin-triggered ROS bursts of mock and estradiol treatment groups, we can determine whether the CERK1 variant is resistant to AvrPtoB (<strong>Fig.14</strong>).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_14.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure14" src="../Result_figures/Figure_14.png" /></a></p>
<figcaption> Figure 14. The assay used to screen for AvrPtoB-resistant CERK1 variants in planta.
</figcaption>
</figure>
<p>Once again, time constraints did not allow us to test variants, but we were able to show that we had successfully silenced the native CERK1 genes. We did this by performing a ROS burst assay, where the silenced plants showed no response to chitin when compared to non-VIGS plants (<strong>Fig. 15</strong>).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_15.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure15" src="../Result_figures/Figure_15.png" /></a></p>
<figcaption> Figure 15. Successful silencing of native CERK1 of N. benthamiana using VIGS.
</figcaption>
</figure>
<p><strong>Conclusion:</strong> We successfully designed a two-round screening approach to rapidly screen for functional and AvrPtoB-resistant CERK1 variants in plants.</p>
<h2 id="development-of-trust-rmyth-a-reverse-split-ubiquitin-based-membrane-protein-yeast-two-hybrid-system-with-e2a-bleor-anti-truncation-strategy">Development of TRUST-rMYTH, a reverse split ubiquitin-based membrane protein yeast two-hybrid system with E2A-BleoR anti-truncation strategy</h2>
<p>As already mentioned in the Background section, aside from the conventional Y2H systems, many genetic methods have been developed to study protein-protein interactions. One popular approach is the split-ubiquitin membrane protein yeast two-hybrid (MYTH). Unfortunately, the system is currently usable only in the forward direction, with no successfully developed reverse variations. Therefore, we strived to develop a reverse MYTH system using the E2A-BleoR strategy to eliminate truncation mutations.</p>
<p>Aim: Develop an assay to disrupt the interaction between membrane proteins and prove that the E2A-BleoR anti-truncation strategy can be extensively applied to different genetic PPI assays.</p>
<p>To allow our novel MYTH system to run in both directions we chose to base it on the <em>Ura3</em> reporter gene (Link to reverse Y2H part in introduction). To that end, we repurposed the FRY1537 yeast strain (Ottoz et al., 2014), which already carries a <em>Ura3</em> gene driven by a synthetic promoter that can be activated by the LexA-VP16 synthetic transcription factor. With this, we had all the components for the rMYTH system, they just required a bit of putting together. To obtain results that are a bit more quantitative than the yeast plate assays, we also constructed a fluorescence-based yeast strain. In the case of protein-protein interaction, this yeast will emit fluorescence, which can be easily measured using a flow cytometer or a plate reader.</p>
<p>As a proof-of-concept, we used two well-studied single-pass membrane proteins in the plant immune systems, BAK1 and BIR2, to test our rMYTH system (<strong>Fig. 16A</strong>). BAK1 and BIR2 have been reported to interact with each other through both extracellular and intracellular domains (Ma et al., 2017; Halter et al., 2014). As the residues on the interaction surfaces were previously identified, we generated mutations on some of these residues to disrupt either extracellular or intracellular interaction or both of them (<strong>Fig. 16B</strong>). </p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_16.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure16" src="../Result_figures/Figure_16.png" /></a></p>
<figcaption> Figure 16. (A) Schematic diagram of the rMYTH design used for selection of loss-of-interaction mutations on the plate with 0.5 mg/mL 5-FOA. (B) BAK1 and BIR2 with interaction-disrupting mutations are used to test the rMYTH system.
</figcaption>
</figure>
<p>Different mutations resulted in BAK1-BIR2 pairs with different levels of interaction strength, which were quantified by the fluorescence readout (<strong>Fig. 17</strong> rightmost bar plot). As shown in the plate assay results (<strong>Fig. 17</strong>), there was a positive correlation between the interaction strength and the cell growth on the plate without uracil, while the 5-FOA plate showed an opposite trend. These results suggest the success of repurposing FRY1537 as a reporter strain for rMYTH.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_17.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure17" src="../Result_figures/Figure_17.png" /></a></p>
<figcaption> Figure 17. E2A-BleoR can be fused after either Nub or Cub-LexA-VP16 without or only mildly affecting BAK1-BIR2 interaction
Differences between sample means in the flow-cytometry data were assessed using one-way ANOVA with Dunnett’s post-hoc test; &lt;0.01 (*), &lt;0.0001(****)
</figcaption>
</figure>
<p>We then tested whether the E2A-BleoR anti-truncation strategy can be also applied to rMYTH (<strong>Fig. 18</strong>) E2A-BleoR and (GS)<sub>3</sub>-BleoR were fused after Cub-LexA-VP16 for the bait construct and after Nub for the prey construct, respectively. The fusions of E2A-BleoR after Nub and Cub-LexA-VP16 had no effect and a neglectable negative effect on BAK1-BIR2 interaction, respectively. For (GS)<sub>3</sub>-BleoR, a 50% decrease of interaction strength was observed when it was fused after the Nub, suggesting that the covalent fusion severely interfered with the detection of interaction. When  (GS)<sub>3</sub>-BleoR was fused after the Cub-LexA-VP16, an increase in the interaction strength was observed, which implies that (GS)<sub>3</sub>-BleoR can cause false positive results, perhaps because of the homodimerization of BleoR (Dumas et al., 1994).</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_18.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure18" src="../Result_figures/Figure_18.png" /></a></p>
<figcaption> Figure 18. Fusion of E2A-BleoR after Nub or Cub-LexA-VP16 has little consequence on interaction
</figcaption>
</figure>
<p><strong>Conclusion:</strong> We successfully developed TRUST-rMYTH, a reverse split-ubiquitin membrane protein yeast two-hybrid system equipped with E2A-BleoR anti-truncation strategy, which can be used to disrupt interactions between membrane-bound proteins while avoiding truncations.</p>
<h2 id="construction-of-fluorescent-reporter-strains-to-quantitatively-measure-interaction-strength">Construction of fluorescent reporter strains to quantitatively measure interaction strength</h2>
<p>In classic Y2H and MYTH assays, the existence of interaction and the difference of interaction strengths are concluded mainly based on the cell growth, which is not quantitative enough, and might not be able to detect the small changes in the interaction strength. Although most commercial strains are equipped with a LacZ reporter gene enabling the usage of enzymatic activity assays (like ONPG assay) to quantify the interaction, it is very labour-intensive to conduct such assays, especially when handling a large number of samples. </p>
<p><strong>Aim:</strong> To develop reporter strains carrying a fluorescent reporter gene for both conventional Y2H and MYTH to precisely measure the interaction strength. </p>
<p>The construction strategies for the fluorescent reporter strains for both the conventional Y2H, as well as  MYTHare shown on <strong>Figure 19</strong>.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_19.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure19" src="../Result_figures/Figure_19.png" /></a></p>
<figcaption> Figure 19. Construction of fluorescent reporter strain OVY216+pGAL1-ymUKG1 for the conventional Y2H, and FRY70+FRP795 for split-ubiquitin membrane protein Y2H.

</figcaption>
</figure>
<p>As shown in Figure 20, the workflow, which is based on liquid handling, is very simple.</p>
<figure>
<p><a class="glightbox" href="../Result_figures/Figure_20.png" data-type="image" data-width="100%" data-height="auto" data-desc-position="bottom"><img alt="Figure20" src="../Result_figures/Figure_20.png" /></a></p>
<figcaption>Figure 20. Workflow to quantitatively measure PPI strength using the fluorescent reporter strains.

</figcaption>
</figure>
<p>These reporter strains have been already successfully used in our research to draw the following conclusions:</p>
<ul>
<li>
<p>In the TRUST-rY2H system, the E2A-BleoR fusion has almost no interference on the CERK1-AvrPtoB interaction, and E2A is the best 2A self-cleavage peptide in comparison with O2A and P2A (<strong>Fig 6</strong>).</p>
</li>
<li>
<p>In yeast, BAK1 and BIR2 interact with each other through both the extracellular and intracellular domains (<strong>Fig. 17</strong>). </p>
</li>
<li>
<p>In the TRUST-rMYTH system, the E2A-BleoR anti-truncation module does not interfere with the interaction detection module (<strong>Fig. 18</strong>).</p>
</li>
</ul>
<p>In the future, we will run a rY2H assay in strains carrying both <em>Ura3</em> and fluorescent reporter genes, so that we can directly pick colonies from 5-FOA plates to measure to what extent the interaction is disrupted.</p>
<h2 id="references">References:</h2>
<p>P. Dumas, M. Bergdoll, C. Cagnon, and J. M. Masson, Crystal structure and site-directed mutagenesis of a bleomycin resistance protein and their significance for drug sequestering. EMBO J, vol. 13, pp. 2483–2492, (1994). DOI: https://10.1002/J.1460-2075.1994.TB06535.X</p>
<p>M. Tominaga, K. Nozaki, D. Umeno, J. Ishii, and A. Kondo, Robust and flexible platform for directed evolution of yeast genetic switches. Nature Communications, vol. 12, pp. 1–13, (2021). DOI: https://10.1038/s41467-021-22134-y</p>
<p>W. Cheng et al., Structural Analysis of Pseudomonas syringae AvrPtoB Bound to Host BAK1 Reveals Two Similar Kinase-Interacting Domains in a Type III Effector. Cell Host Microbe, vol. 10, pp. 616–626, (2011). DOI: https://10.1016/J.CHOM.2011.10.013</p>
<p>A. Pietraszewska-Bogiel et al., Interaction of Medicago truncatula Lysin Motif Receptor-Like Kinases, NFP and LYK3, Produced in Nicotiana benthamiana Induces Defence-Like Responses. PLoS One, vol. 8, pp. 65055, (2013). DOI: https://10.1371/JOURNAL.PONE.0065055</p>
<p>C. Ma et al., Structural basis for BIR1-mediated negative regulation of plant immunity. Cell Research, vol. 27, pp. 1521–1524, (2017). DOI: https://10.1038/cr.2017.123</p>
<p>T. Halter et al., The leucine-rich repeat receptor kinase BIR2 is a negative regulator of BAK1 in plant immunity. Current Biology, vol. 24, pp. 134–143, (2014). DOI: https://10.1016/j.cub.2013.11.047</p>





                
              </article>
            </div>
          
          
        </div>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
    <div class="md-copyright__highlight">
      Copyright &copy; 2023 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
    </div>
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "../..", "features": ["navigation.sections", "navigation.instant", "navigation.tabs"], "search": "../../assets/javascripts/workers/search.f886a092.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    
      <script src="../../assets/javascripts/bundle.94c44541.min.js"></script>
      
        <script src="../../javascripts/config.js"></script>
      
        <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"></script>
      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.2.1/tablesort.min.js"></script>
      
        <script src="../../javascripts/tables.js"></script>
      
    
  <script>document$.subscribe(() => {const lightbox = GLightbox({"touchNavigation": true, "loop": false, "zoomable": true, "draggable": true, "openEffect": "zoom", "closeEffect": "zoom", "slideEffect": "slide"});})</script></body>
</html>